# SerialData.msg - 动态串口数据消息定义
Header header

# 命名数据字段数组，每个元素包含字段名和对应的值
Name_Value[] data_fields

float64 sec_f

